### ISSUE TITLE
[Bug]: Broken CJK/Asian font rendering

### ISSUE THREAD

**Issue by RotaryBoot58** [FreeTubeApp/FreeTube#4142]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1-Open Freetube
2-Open a video that has CJK/Asian characters (i.e. https://youtu.be/6j8EFfigkMg)
3-The characters are a square with a X in the middle
![Screenshot_20231013_002536](https://github.com/FreeTubeApp/FreeTube/assets/104520911/7c89df34-b7ae-41be-a6a2-9243ce732fe3)

### Expected Behavior

CJK/Asian to be rendered properly

### Issue Labels

text/string issue, visual bug

### FreeTube Version

v0.19.1

### Operating System Version

openSUSE Tumbleweed

### Installation Method

Flathub

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.19.0

### Additional Information

Operating System: openSUSE Tumbleweed 20231006
KDE Plasma Version: 5.27.8
KDE Frameworks Version: 5.110.0
Qt Version: 5.15.10
Kernel Version: 6.5.4-1-default (64-bit)
Graphics Platform: Wayland
Processors: 12 × AMD Ryzen 5 3600 6-Core Processor
Memory: 11.6 GiB of RAM
Graphics Processor: AMD Radeon RX 570 Series
Manufacturer: Gigabyte Technology Co., Ltd.
Product Name: B450 AORUS M

running the app from the terminal:
`F: Not sharing "/usr/share/icons" with sandbox: Path "/usr" is reserved by Flatpak
LaunchProcess: failed to execvp:
xdg-settings
[2:1013/003544.262108:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:1013/003544.262176:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[46:1013/003548.372662:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[46:1013/003607.991656:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[46:1013/003610.194738:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
[46:1013/003612.378989:ERROR:vaapi_video_decoder.cc(1206)] failed Initialize()ing the frame pool
[46:1013/003612.385247:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RED_8, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.385381:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RG_88, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.387390:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RED_8, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.387487:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RG_88, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.389272:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RED_8, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.389352:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RG_88, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.391100:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RED_8, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.391181:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RG_88, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.392631:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RED_8, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003612.392702:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RG_88, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003613.094753:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RED_8, share_between_threads: 0, gmb_type: shared_memory
[46:1013/003613.094894:ERROR:shared_image_factory.cc(575)] Could not find SharedImageBackingFactory with params: usage: Gles2|Raster|DisplayRead|Scanout, format: RG_88, share_between_threads: 0, gmb_type: shared_memory
`

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by Adecom16** [comment]
Can you assign this to me and receive an instant PR

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @RotaryBoot58 do you maybe have other example videos i cant reproduce this issue on my windows machine and on my fedora machine

**Comment by RotaryBoot58** [comment]
> Hi @RotaryBoot58 do you maybe have other example videos i cant reproduce this issue on my windows machine and on my fedora machine

Yes, plenty of them, here is the list of some videos:
https://youtu.be/QslJYDX3o8
https://youtu.be/f6TytcA47rI
https://youtu.be/kGDWVV_1do4
https://youtu.be/FDqUDoOhKoA

![Screenshot_20231015_003031](https://github.com/FreeTubeApp/FreeTube/assets/104520911/ed82af89-a778-415e-853e-7e45fa92160d)

> i cant reproduce this issue on my windows machine and on my fedora machine

That is strange, I logged into a system snapshot using version 0.19.0, and everything was working fine (no system update were done) and for some reason, it is happening on my openSUSE and EndeavourOS machines.

The only change that I made was running a flatpak update, can it be a bug related with freedesktop platform or something else?

**Comment by RotaryBoot58** [comment]
So, apparently the font issue is only happening with the flatpak package, I tested the snap and rpm packages and everything was working just fine.

The workaround I find was to give permission to acess the fonts directory and installing the google-noto-sans* packages for proper rendering (even though only freetube was having this issue).

A friend of mine was able to reproduce this bug and fix it the same way I did, I don't really know if it is something related to freetube or flatpak itself, but all I can say is that this is the way I find to fix this bug and hope it can help somebody else going through the same problem.
